### PR TITLE
german translation (informal)

### PR DESCRIPTION
**Pull request by uDEV2019**
changed from formal to informal

What is not changed (opposite informal translation):
- "Change Type" for custom fields [because it makes even in english no sense, when you press the gearwheel (Change-Button)]
- and two other strings from discussion on last PR

### CODE REVIEW

File: src/i18n/translations/de.json [hunk 1]
```diff
@@ -38,64 +38,83 @@
...
+    },
     "none-selected": {
-      "message": "Dokument auswählen oder bearbeiten",
+      "message": "Auswählen oder Erstellen eines neuen Dokuments",
       "title": "Kein Dokument ausgewählt"
     },
     "save": "Speichern",
+    "save-disabled-title": {
+      "readonly": "Bearbeiten ist derzeit nicht möglich",
+      "untitled": "Einige Felder wurde nicht benannt"
```
Comment (julianpoemp) [inline src/i18n/translations/de.json:64]: typo: use "wurden" instead of "wurde"

File: src/i18n/translations/de.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
       "cancel": "Abbrechen",
       "cancel-title": "Löschen von Anhang abbrechen",
       "delete": "Löschen",
-      "delete-prompt": "Das Löschen dieses Anhangs wird ihn permanent aus dem Tresor entfernen.\nSind Sie sicher, dass Sie diesen löschen möchten?",
+      "delete-prompt": "Das Löschen dieses Anhangs wird ihn permanent aus dem Tresor entfernen.\n Bist Du sicher, dass Du diesen löschen möchtest?",
```
Comment (julianpoemp) [inline src/i18n/translations/de.json:7]: Please remove the empty space after the newline "\n".

**Review by julianpoemp — COMMENTED** [review]
The translation looks good! There are just two small things that need to be changed, but after that we can use it :)

**Review by julianpoemp — APPROVED** [review]
looks good! 👍

**Review by perry-mitchell — APPROVED** [review]

**Pull request merged**